src/ResourceManager/Sql/Commands.Sql/SqlDatabaseAgent/Model/AzureSqlDatabaseAgentJobCredentialModel.cs

```diff
@@ -0,0 +1,59 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System.Security;
+
+namespace Microsoft.Azure.Commands.Sql.SqlDatabaseAgent.Model
+{
+    /// <summary>
+    /// Represents the core properties of a job credential
+    /// </summary>
+    public class AzureSqlDatabaseAgentJobCredentialModel
```

Comment:
It would be nice to add a "PS" prefix to each of these models to indicate that is a PowerShell type instead of an SDK type.